src/sysoptions.h

```diff
@@ -264,6 +264,8 @@
 #if DROPBEAR_SNTRUP761
 /* 2337 */
 #define MAX_KEX_PARTS (2*4 + 1158 + 1039 + 32*2 + 68)
+#elif DROPBEAR_MLKEM768
+#define MAX_KEX_PARTS (2*4 + 1184 + 1088 + 32*2 + 68)
```

Comment:
`MLKEM768` is larger than `SNTRUP761` so it should be first in the `#if` sequence.

Reply:
Fixed.